docs/learn/fundamentals/stellar-data-structures/contracts.mdx

```diff
@@ -5,17 +5,17 @@ sidebar_position: 50
 
 :::note
 
-Stellar has integrated a smart contracts platform called "[Soroban](../../../build/smart-contracts/getting-started/README.mdx)" into the core protocol.
+Stellar has integrated a smart contracts platform called "Soroban" into the core protocol. Write a smart contract on Stellar by following this [Getting Started guide](../../../build/smart-contracts/getting-started/README.mdx).
```

Comment:
(Not a comment I will fight for lol)

Maybe "Getting Started tutorial" at the end here? "guide" (in my head) is more of the "bite-sized" how-to things we've got going on, and "tutorial" would be more reflective of the getting started docs.